Mathlib/Data/Nat/Digits.lean

```diff
@@ -115,12 +115,16 @@ theorem digits_one_succ (n : ℕ) : digits 1 (n + 1) = 1 :: digits 1 n :=
   rfl
 #align nat.digits_one_succ Nat.digits_one_succ
 
-@[simp]
 theorem digits_add_two_add_one (b n : ℕ) :
     digits (b + 2) (n + 1) = ((n + 1) % (b + 2)) :: digits (b + 2) ((n + 1) / (b + 2)) := by
   simp [digits, digitsAux_def]
 #align nat.digits_add_two_add_one Nat.digits_add_two_add_one
 
+@[simp]
+lemma digits_of_two_le_of_pos (hm : 2 ≤ m) (hn : 0 < n) :
```

Comment:
Perhaps `m` should be renamed to `b`, for consistency with the other declarations in this file?

Reply:
Moreover, currently `m` is an autoimplicit, which I think goes against general mathlib style.